src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/files/main.js

```diff
@@ -237,18 +242,108 @@ define(function () {
         },
 
         /**
+         * render move media overlay
+         */
+        renderSelectCollection: function() {
+            this.sandbox.start([{
+                name: 'collections/components/collection-select@sulumedia',
+                options: {
+                    el: this.$find(constants.moveSelector)
+                }
+            }]);
+        },
+
+        /**
+         * starts overlay for move media
+         */
+        startMoveMediaOverlay: function() {
+            this.sandbox.emit('husky.datagrid.items.get-selected', function(ids) {
+                this.selectedIds = ids;
+                this.sandbox.emit('sulu.media.collection-select.open');
+            }.bind(this));
+        },
+
+        /**
+         * emit events to move selected media's
+         * @param collection
+         */
+        moveMedia: function(collection) {
+            var left = this.selectedIds.length;
+
+            this.sandbox.emit('sulu.media.collections.move-media', this.selectedIds, collection,
```

Comment:
the name comes from `MOVE_MEDIA` from the collection component which has the namespace `           namespace = 'sulu.media.collections.',` https://github.com/sulu-cmf/sulu/pull/805/files#diff-c7c75c1dba09b1dbc3184df6b64ef95fR306